crates/avro/src/snapshots/avro__encode__test__basic_types-2.snap

```diff
@@ -14,14 +14,14 @@ expression: to_hex(&b)
  |00000300 00006568 e0000080 80f3bbb5| ......eh........ 00000090
  |d8c10500 48613062 30633064 302d3130| ....Ha0b0c0d0-10 000000a0
  |30302d32 3030302d 33303030 2d343030| 00-2000-3000-400 000000b0
- |30303030 30303030 30142272 6177206a| 000000000."raw j 000000c0
- |736f6e22 00020a6a 656e6e79 daffa208| son"...jenny.... 000000d0
- |00000401 00000802 02000001 02020261| ...............a 000000e0
- |001e226c 65616469 6e672065 78747261| .."leading extra 000000f0
- |22000001 00010202 65001c22 6d696464| ".......e.."midd 00000100
- |6c652065 78747261 22000200 0102027a| le extra"......z 00000110
- |00202274 7261696c 696e6720 65787472| . "trailing extr 00000120
- |61220000 0002126e 6f742076 616c6964| a".....not valid 00000130
- |01000000 01020e7a 7a202d20 65780016| .......zz - ex.. 00000140
- |5b226578 74726122 2c335d00|          ["extra",3].     00000150
-                                                        0000015c
+ |30303030 30303030 30000214 22726177| 000000000..."raw 000000c0
```

Comment:
@jgraettinger im not entirely sure how to verify that this snapshot update is correct. The other 2 are exactly what I would expect to see, so I'm pretty sure this one is too, but figured I'd check before hitting merge.

Reply:
Yep I think it's good.